ios/Rainbow/AppDelegate.mm

```diff
@@ -179,8 +183,20 @@ - (void)applicationDidBecomeActive:(UIApplication *)application{
   }
   // delete the badge
   [UIApplication sharedApplication].applicationIconBadgeNumber = 0;
-  // delete the notifications
-  [[UNUserNotificationCenter currentNotificationCenter] removeAllDeliveredNotifications];
+  // delete the notifications from WC
```

Comment:
Changes behavior, to clear only wallet connect notifications and leave all other notifications. Before we supported only wallet connect notifications and we were clearing all of the notifications